WikipediaUnitTests/Code/ArticleFetcherTests.m

```diff
@@ -47,6 +47,10 @@ - (void)tearDown {
     [super tearDown];
 }
 
++ (NSArray<NSInvocation*>*)testInvocations {
+    return WMFIsTravis() ? @[] : [super testInvocations];
```

Comment:
@bgerstle oh so that is on XCTest? I didn't realize that you could prevent the test from running like that - cool.